### PR TITLE
Andy edits

### PR DESCRIPTION
**Pull request by ateucher**

### CODE REVIEW

**Comment by ateucher** [comment]
Hey @stephhazlitt @boshek @gcperk some eyes on this would be great if you can. At least for the general direction, then we can merge it so the structure is up to date, then I can continue on with the content...

File: 11-rmarkdown-application-regression.Rmd [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+&#96;&#96;&#96;{r model}
+canada_lm <- lm(gdpPercap ~ lifeExp, data = canada)
+summary(canada_lm)
+&#96;&#96;&#96;
+</pre>
+
+The `summary()` function gives us the high level information from our regression 
+model: The coefficient estimates, the R-squared, and the p-value.
+
+And the `plot()` function gives us some basic diagnostic plots:
```
Comment (boshek) [inline 11-rmarkdown-application-regression.Rmd:83]: I _think_ this is the first time we mentioned plot so you might give a little preamble.

File: 11-rmarkdown-application-regression.Rmd [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+plot(canada_lm)
+&#96;&#96;&#96;
+</pre>
+
+But these `lm` objects are ungainly - they are fine to look at to inspect the 
+important parts of our model, but difficult to extract those pieces that we 
+want to use later - such as regression coefficients and P values.
+
+There is another package we can use, called *broom*, that is used to tidy
+model results into a format we are familiar with: A `data.frame`:
```
Comment (boshek) [inline 11-rmarkdown-application-regression.Rmd:96]:
```suggestion
model results into a format we are familiar with: a `data.frame`. 
```
Also is it worth mentioning that broom works on many types of models?

File: 11-rmarkdown-application-regression.Rmd [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+Working in a new R Markdown document (File -> New File -> R Markdown Document), 
+we can load the libraries that we are going to use, and load the data in 
+a code chunk:
+
+<pre>
+&#96;&#96;&#96;{r load}
+library(readr)
+library(dplyr)
+library(ggplot2)
+gapminder <- read_csv("data/gapminder_data.csv")
```
Comment (gcperk) [inline 11-rmarkdown-application-regression.Rmd:35]: It looks like you are using both read_csv and read.csv - did we come to a standard? Sam and I are both using read_csv in the R Markdown lesson (10)

File: 11-rmarkdown-application-regression.Rmd (hunk 1 above)
Comment (ateucher) [reply 11-rmarkdown-application-regression.Rmd:83]: Check, good call

File: 11-rmarkdown-application-regression.Rmd (hunk 2 above)
Comment (ateucher) [reply 11-rmarkdown-application-regression.Rmd:96]: 👍

File: 11-rmarkdown-application-regression.Rmd (hunk 3 above)
Comment (ateucher) [reply 11-rmarkdown-application-regression.Rmd:35]: I think `read_csv` is the consensus. I don't think I used `read.csv` in this lesson, but it is in one of the challenges in lesson 5 - which I think I'm going to change

**Pull request merged**